entries/g/giphy.com.json

```diff
@@ -4,7 +4,8 @@
     "contact": {
       "twitter": "GIPHY",
       "facebook": "GIPHY",
-      "email": "support@giphy.com"
+      "email": "support@giphy.com",
+      "form": "https://support.giphy.com/hc/en-us/requests/new?ticket_form_id=360000249791"
```

Comment:
Per [the schema](https://github.com/2factorauth/twofactorauth/blob/master/tests/schema.json#L103-L118), only `email` or `form` can be specified, not both. Which would be the best to keep?